CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+# [1.0.5] - 2025-02-10
+
+### Features
+* Support a configurable readiness check endpoint via HTTP.
+
+### Enhancements
+* Record granular spanner api called in request count and latency metrics.
```

Comment:
Add label "spanner_api" in request count and latency metrics for granular observability